packages/client/components/syntax-highlight/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import SyntaxHighlighter from 'react-syntax-highlighter'
+import {
+  vs2015,
+  a11yLight
+} from 'react-syntax-highlighter/dist/cjs/styles/hljs'
+
+export enum ColorTheme {
+  DARK = vs2015,
+  LIGHT = a11yLight
+}
+
+export interface Props {
+  codeString: string
+  highlightedLines?: [number?]
```

Comment:
Why can each element be undefined?

Reply:
> Why can each element be undefined?

the default value of is `highlightedLines = []`. this is not a type of `[number]`

Reply:
Why is it `[number]` instead of `number[]`, `[number]` suggest it's a list with exactly one element.

Reply:
Interesting now `[]` can be set into `[number?]`, I guess it's because `[][0] === undefined`.